handler/src/main/java/io/netty/handler/address/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2019 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Package to dynamically replace local / remote {@link java.net.SocketAddress}.
```

Comment:
Package to dynamically replace -> Allows replacing ... dynamically when making a connection attempt. ?

Reply:
I am not sure... I was thinking maybe at some point we also have something for bind here.

Reply:
OK.